### ISSUE TITLE
A11y : Cannot use FileUpload component via keyboard

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46501]
**Steps to reproduce**:

[FileUpload component](https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/FileUpload/FileUpload.tsx#L45-L56): Try to click "Upload JSON file" button inside import modal

**Actual Result**:

not accesible via keyboard

**Expected Result**

workflow accessible for keyboard-only users

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:
